vminitd/Sources/vminitd/ManagedProcess.swift

```diff
@@ -144,130 +149,153 @@ final class ManagedProcess: Sendable {
         self.terminal = stdio.terminal
         self.bundle = bundle
         self.state = Mutex(State(io: io))
+        self.state.withLock { state in
+            state.errorPipe = errorPipe
+        }
```

Comment:
Just add a new errorPipe field to the constructor so we can do `Mutex(State(io: io, errorPipe: errorPipe))` instead of this.